openslides/core/models.py

```diff
@@ -243,3 +273,61 @@ class Meta:
     def get_session_store_class(cls):
         from .session_backend import SessionStore
         return SessionStore
+
+
+class Message(RESTModelMixin, models.Model):
```

Comment:
Yes. The other way would be to transform the messages and countdowns to real root rest models and remove the extra view points in the projector view set. But I like the current way a little bit more: Just save messages into projector as it is in master.